rules/integrations/aws_bedrock/aws_bedrock_multiple_validation_exception_errors_by_single_user.toml

```
@@ -29,6 +29,39 @@ references = [
 ]
 risk_score = 73
 rule_id = "725a048a-88c5-4fc7-8677-a44fc0031822"
+note = """## Triage and analysis
+
+### Investigating Amazon Bedrock Model Validation Exception Errors.
+
+Amazon Bedrock is AWS’s managed service that enables developers to build and scale generative AI applications using large foundation models (FMs) from top providers.
+
+Bedrock offers a variety of pretrained models from Amazon (such as the Titan series), as well as models from providers like Anthropic, Meta, Cohere, and AI21 Labs.
+
+#### Possible investigation steps
+
+- Identify the user account that caused validation errors in accessing the Amazon Bedrock models.
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Consider the time of day. If the user is a human (not a program or script), did the activity take place during a normal time of day?
+- Examine the account's attempts to access Amazon Bedrock models in the last 24 hours.
+- If you suspect the account has been compromised, scope potentially compromised assets by tracking Aamzon Bedrock model access, prompts generated, and responses to the prompts by the account in the last 24 hours.
```

```suggestion
- If you suspect the account has been compromised, scope potentially compromised assets by tracking Amazon Bedrock model access, prompts generated, and responses to the prompts by the account in the last 24 hours.
```

Can you fix this for the others

---

Done